### ISSUE TITLE
pretrain_collate_fn

### ISSUE THREAD

**Issue by Towardfront** [qibin0506/llm_trainer#4]
你好，请问llm_trainer\llm_trainer\utils.py，pretrain_collate_fn函数
def pretrain_collate_fn(batch_data):
    # [[x,x,x], [y,y,y]]
    inputs = pad_sequence(batch_data, batch_first=True, padding_value=TrainerTools().tokenizer.pad)
    # crossEntropy默认的ignore_index是-100
    labels = pad_sequence(batch_data, batch_first=True, padding_value=-100)

    # inputs, labels
    return {
        'inputs': inputs,
        'labels': labels
    }
处理label时是否没有移位？
